src/util/CloneEquals.ts

```diff
@@ -1,3 +1,4 @@
+import JSType from "const/JSType";
```

Comment:
We don't want to make any modifications to this file, as it is the subset of things that we derived from Underscore under the defined license terms.  Should they make a critical bug fix, we're going to want to migrate copies in without having to guess at the delta.

Reply:
reverted